src/lib/connectors/bookshelf/ChargeVersion.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const { bookshelf } = require('./bookshelf.js');
+
+module.exports = bookshelf.model('ChargeVersion', {
+  tableName: 'charge_versions',
+
+  hasTimestamps: ['date_created', 'date_updated'],
+
+  idAttribute: 'charge_version_id',
+
+  chargeElements () {
+    return this.hasMany('ChargeElement', 'charge_version_id', 'charge_version_id');
```

Comment:
'charge_version_id' has been added twice.

Reply:
Once is for the local table and once for the foreign key - I think this is OK